TSQLLINT_CONSOLE/Application.cs

```diff
@@ -5,36 +5,34 @@ namespace TSQLLINT_CONSOLE
 {
     public class Application
     {
-        private readonly string[] Args;
-        private readonly IReporter Reporter;
-        private readonly ConsoleTimer Timer = new ConsoleTimer();
+        private readonly string[] _args;
+        private readonly IReporter _reporter;
+        private readonly ConsoleTimer _timer = new ConsoleTimer();
 
         public Application(string[] args, IReporter reporter)
         {
-            Timer.start();
-            Args = args;
-            Reporter = reporter;
+            _timer.Start();
+            _args = args;
+            _reporter = reporter;
         }
 
         public void Run()
         {
             // parse options
-            var commandLineOptions = new CommandLineOptions(Args);
+            var commandLineOptions = new CommandLineOptions(_args);
 
             // perform non-linting actions
-            var configHandler = new ConfigHandler.ConfigHandler(commandLineOptions, Reporter);
-            configHandler.HandleConfigs();
-
-            if (!configHandler.PerformLinting)
+            var configHandler = new ConfigHandler.ConfigHandler(commandLineOptions, _reporter);
+            if (!configHandler.HandleConfigs())
```

Comment:
I did change HandleConfigs to return the success instead of using a Property on the object for this purpose